docs/transparent-validation/msi-z790/laboratory-assembly-guide.md

```diff
@@ -303,3 +303,8 @@ Example output:
 
 * `1` means that the platform is turned on.
 * `0` means that the platform is turned off.
+
+### USB devices
+
+Since some issues with USB controllers may only happen on select USB ports,
```

Comment:
Do Z690 and Z790 really have different counts of USB ports? Or is it because they have different adapters plugged into the USB headers?

Reply:
At least that seems the case from https://www.msi.com/Motherboard/PRO-Z790-P-WIFI/Specification and https://www.msi.com/Motherboard/PRO-Z690-A-WIFI/Specification although it looks like I made a mistake and Z790 has 25 ports, but Z690 still has 17.

Reply:
Huh, interesting. Thanks